apps/web/playwright/google-calendar.e2e.ts

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+import { expect } from "@playwright/test";
+
+import { test } from "./lib/fixtures";
+import { installGoogleCalendar } from "./lib/testUtils";
+
+test.describe.configure({ mode: "parallel" });
+
+const GOOGLE_CALENDAR_EMAIL = process.env.E2E_TEST_GOOGLE_CALENDAR_EMAIL!;
+const GOOGLE_CALENDAR_PASSWORD = process.env.E2E_TEST_GOOGLE_CALENDAR_PASSWORD!;
+const GOOGLE_APPNAME = process.env.E2E_TEST_GOOGLE_AUTH_CONSENT_APPNAME!;
+
+const SHOULD_SKIP_TESTS = !GOOGLE_CALENDAR_EMAIL || !GOOGLE_CALENDAR_PASSWORD || !GOOGLE_APPNAME;
```

Comment:
I actually think we should fail tests instead of skipping them. This could be easily overlooked and never fixed.

Reply:
Yeah I think that's a good idea, but with a check of HOSTED_CAL so that they are required for us only and doesn't fail for Self Hosters.

Reply:
Agreed with both!

Reply:
We have other places (I can think of Apple Calendar and Stripe off the top of my head) where the approach is the same: skipping tests rather than failing them. IDK if this will be tackled on the issues, but we should fix it :eyes:

Reply:
Added the requested changes.